scylla/src/transport/cql_types_test.rs

```diff
@@ -116,9 +116,103 @@ async fn test_varint() {
         "-129",
         "123456789012345678901234567890",
         "-123456789012345678901234567890",
```

Comment:
You could add a test with bigger number, that can't be contained in any primitive type (this number is in range of i128).